### ISSUE TITLE
replication_test.py::test_flushall_in_full_sync

### ISSUE THREAD

**Issue by kostasrim** [dragonflydb/dragonfly#3897]
https://github.com/dragonflydb/dragonfly/actions/runs/11247452053/job/31270963227

```
      # Make sure that a new sync ID is present, meaning replication restarted following FLUSHALL.
        new_syncid, _ = await c_replica.execute_command("DEBUG REPLICA OFFSET")
>       assert new_syncid != syncid
E       AssertionError: assert 'SYNC1' != 'SYNC1'
```

**Comment by kostasrim** [comment]
also this one https://github.com/dragonflydb/dragonfly/actions/runs/11243645404/job/31260022992#step:6:853

**Comment by kostasrim** [comment]
Fixed in #3929
